recipes/keras-tuner/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "keras-tuner" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f018ded5127e7c588bd78faa753fa1f4108f1f28c0e5694a06bfb1f08204215b"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - colorama
+    - future
+    - numpy
+    - pip
+    - python
+    - requests
+    - scikit-learn
+    - scipy
+    - tabulate
+    - terminaltables
+    - tqdm
+  run:
```

Comment:
It looks like you're going to need `tensorflow`:

https://github.com/keras-team/keras-tuner/blob/master/setup.py#L33

Reply:
Yes, you are right. And that was the main reason why I stopped fixing the recipe, there is no tensorflow 2.0 on conda-forge yet, and i did not have time to look into that yet. Is there any problem or some other constraints why tensorflow is still on v1.13.2 on conda-forge?